config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -92,7 +92,7 @@ periodics:
     path_alias: k8s.io/release
   spec:
     containers:
-    - image: ubuntu:latest
+    - image: quay.io/k8s/release-tools:latest
```

Comment:
something something staging registries? /shrug

Reply:
@BenTheElder -- already talking with k8s-infra about staging repos; we're just trying to tweak the permissions for GCR first. :)